cogs/tools.py

```diff
@@ -47,21 +46,6 @@ async def ask_question(self, ctx: Context, conversation_mode: str = "true", *, q
 
         await ctx.send(embed=embed)
 
-    @command()
-    async def charinfo(self, ctx: Context, *, characters: str) -> t.Union[str, None]:
-        """Shows you information about up to 25 characters."""
-
-        def to_string(c: str) -> str:
-            digit = f"{ord(c):x}"
-            name = unicodedata.name(c, "Name not found.")
-            return f"`\\U{digit:>08}`: {name} - {c} \N{EM DASH} <http://www.fileformat.info/info/unicode/char/{digit}>"
-
-        msg = "\n".join(map(to_string, characters))
-        if len(msg) > 2000:
-            await ctx.send("Output too long to display.")
-            return
-        await ctx.send(msg)
-
     @command()
     async def charinfo_uni(self, ctx: Context, *, characters: str) -> None:
```

Comment:
Since `charinfo` is removed, this should be renamed to `charinfo` not `charinfo_uni`, if you need `charinfo_uni` you can make an alias to the command